### PR TITLE
TILA-1779 | Change allocatedDay to int

### PR DESCRIPTION
**Pull request by nicobav**
Previously enum

TILA-1779

### CODE REVIEW

**Review by saaste — APPROVED** [review]
Coooool stuff! 🥇 Just one suggestion, I leave it up to you 😄

File: api/graphql/applications/application_types.py
Comment (nicobav) [reply api/graphql/applications/application_types.py:16]: Good point, it might be un-needed. Personally I'd like to have non-translated as well defined here. What we really could do is refactor to have only one DAYS since we have it elsewhere also reproduced.
So shortly I'll merge this as is and hopefully when doing enum refactoring to graphql api we can sort this one out too.

**Pull request merged**
